src/main/flight/alt_hold.c

```diff
@@ -174,44 +174,44 @@ void altHoldUpdate(altHoldState_s* altHoldState)
 {
     altHoldProcessTransitions(altHoldState);
 
-    float timeInterval = 1.0f / ALTHOLD_TASK_PERIOD;
+    if (altHoldState->altHoldEnabled) {
```

Comment:
It is not that simple. IIRC althold is tracking quad state even when not enabled. But there were output variables that were not necessary ...